src/functTest/groovy/com/bmuschko/gradle/docker/AbstractFunctionalTest.groovy

```diff
@@ -15,6 +15,7 @@
  */
 package com.bmuschko.gradle.docker
 
+import org.gradle.internal.impldep.com.google.common.io.Files
```

Comment:
Can we remove the use of using an internal call? There are other more OOTB ways to do this very thing which do not require us to use something internal which may/may-not change and/or be removed.